en/complete_vehicles/px4_vision_kit.md

```diff
@@ -0,0 +1,295 @@
+## PX4 Vision Autonomy Development Kit (Early Adopter Edition)
+
+The *PX4 Vision Autonomy DevKit* is a robust and inexpensive kit for enabling computer vision development on autonomous vehicles.
+
+![Overview](../../assets/hardware/px4_vision_devkit/px4_vision_kit_hero.jpg)
+
+The kit contains a near-ready-to-fly carbon-fiber quadcopter equipped with a *Pixhawk 4* flight controller, *UP Core* companion computer, and Occipital *Structure Core* depth camera sensor.
+The vehicle comes with software for PX4 [Obstacle Avoidance](../computer_vision/obstacle_avoidance.md) and [Collision Prevention](../computer_vision/collision_prevention.md) enabled “out of the box”.
+Developers can use the kit to try out other features provided by the PX4 Avoidance project, modify the existing code, or experiment with completely new computer vision-based functionality.
+
+The guide explains the minimal additional setup required to get the vehicle ready to fly (installing an RC system and battery).
+It also covers the first flight, and how to get started with modifying the computer vision code.
+
+
+## Warnings & Notifications
+
+1. Do not power the *UP Core* computer using the power socket (this may destroy the computer).
+   The *UP Core* should only be powered using the battery.
+
+   ![Warning - do not connect power port](../../assets/hardware/px4_vision_devkit/warning_power_port.png)
+
+1. The kit is intended for computer vision projects that use a forward-facing camera (it does not have downward or rear-facing depth cameras).
+   Consequently it can’t be used for testing [Safe Landing](../computer_vision/safe_landing.md), or other features that require a downward-facing camera.
+   
+1. PX4 obstacle avoidance and collision prevention can only be used when GPS is available (not indoors).
+
+
+## Getting to Know Your DevKit
```

Comment:
@jinger26 Note, I combined the schematic and getting to know your devkit sections.Would be good to have a bigger image of the second image below. Do e have source?

Reply:
@jinger26 ^^^^